docs/elements.md

```diff
@@ -1032,33 +1032,33 @@ def generate(data):
-| `placeholder`                | string              | "symbolic expression" | Hint displayed inside the input box describing the expected type of input.                                                                                                                                                                                                                                                      |
-| `custom-functions`           | string              | -                     | A comma-delimited list of custom functions that can be used in the symbolic expression.                                                                                                                                                                                                                                         |
-| `show-score`                 | boolean             | true                  | Whether to show the score badge next to this element.                                                                                                                                                                                                                                                                           |
-| `suffix`                     | text                | -                     | A suffix to display after the input box (e.g., `suffix="$\rm m/s^2$"`).                                                                                                                                                                                                                                                         |
+| Attribute                    | Type                    | Default                 | Description                                                                                                                                                                                                                                                                                                                     |
+| ---------------------------- | ----------------------- | ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `answers-name`               | string                  | —                       | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question. If the correct answer `ans` is a `sympy` object, you should use `import prairielearn as pl` and `data["correct_answers"][answers-name] = pl.to_json(ans)`. |
```

Comment:
Similarly, do we want it to say here: If the correct answer `"and"` is a `"sympy"` object?